config/runtime/boot/fastboot.jinja2

```diff
@@ -0,0 +1,29 @@
+- deploy:
+    images:
+      image:
+        url: '{{ node.artifacts.kernel }}'
+      dtb:
+        url: '{{ node.artifacts.dtb }}'
+      ramdisk:
+        url: 'http://storage.kernelci.org/images/rootfs/buildroot/buildroot-baseline/20230703.0/{{ brarch }}/rootfs.cpio.gz'
+     postprocess:
+      docker:
+        image: ghcr.io/mwasilew/docker-mkbootimage:master
+        steps:
+        - mkbootimg --header_version 2 --kernel Image --dtb qcs6490-rb3gen2.dtb --cmdline "earlycon clk_ignore_unused pd_ignore_unused audit=0" --ramdisk rootfs.cpio.gz --output boot.img
+    to: downloads
+
+- deploy:
+    images:
+      boot:
+        url: 'downloads://boot.img'
+    timeout:
+      minutes: 20
+    to: download
+
+- boot:
+    prompts:
+    - root@(.*):[/~]#
```

Comment:
```suggestion
    - '/ #'
```

As mentioned in https://github.com/kernelci/kernelci-pipeline/pull/623#issuecomment-2139635297

Reply:
yes, this fixes it. Thanks!